tests/api/test_project.py

```diff
@@ -1538,6 +1541,92 @@ def mock_bucketnotfounderror():
             )
             assert response.status_code == http.HTTPStatus.INTERNAL_SERVER_ERROR
             assert "Server Error: Status was not updated" in response.json["message"]
+            assert project.is_active
+
+
+def test_projectstatus_failed_delete_project_content(module_client, boto3_session):
+    """Mock the different expections that can occur when deleting project."""
+    current_unit_admins = models.UnitUser.query.filter_by(unit_id=1, is_admin=True).count()
+    if current_unit_admins < 3:
+        create_unit_admins(num_admins=2)
+    current_unit_admins = models.UnitUser.query.filter_by(unit_id=1, is_admin=True).count()
+    assert current_unit_admins >= 3
+
+    response = module_client.post(
+        tests.DDSEndpoint.PROJECT_CREATE,
+        headers=tests.UserAuth(tests.USER_CREDENTIALS["unituser"]).token(module_client),
+        json=proj_data,
+    )
+    assert response.status_code == http.HTTPStatus.OK
+    project_id = response.json.get("project_id")
+    project = project_row(project_id=project_id)
+
+    def mock_deletionerror():
```

Comment:
Would be good if we could put these outside of the test, at the top of the file, so that more tests could use the same thing. But I remember that being a problem sometimes due to circular imports or something so if it doesn't work then it doesn't. Also I think some of these errors are already implemented somewhere in the test files, but maybe you encountered the circular imports there too